### ISSUE TITLE
GTEx affy

### ISSUE THREAD

**Issue by mimiblis** [BgeeDB/expression-annotations#105]

**Comment by ANiknejad** [comment]
http://bioconductor.org/packages/release/data/annotation/html/pd.hugene.1.1.st.v1.html

should be the Bioconductor link for the platform used in Affy GTEx, see #43

**Comment by mimiblis** [comment]
:+1:

**Comment by ANiknejad** [comment]
please use SRX to label GTEx sample
use SRP012682 to label the GTEx experiment (GSE45878 in GEO)
use (so far) GPL16977 for the platform
put '2' and '2' for normalization and detection

**Comment by mimiblis** [comment]
There is no SRX for affy data

**Comment by fbastian** [comment]
Yep, affy data are not hosted on SRA. You should provide the GSM and GSE IDs.

**Comment by ANiknejad** [comment]
yes, sorry!

**Comment by ANiknejad** [comment]
so Angélique, yes please use GSE and GSM to report Affy samples annotated from our files, not only from the GEO web site (use GEO to retrieve the GSM..if available for all samples)

**Comment by mimiblis** [comment]
Hopefully it wont be the same crap for conversion... of gtex id to gsm cross fingers

**Comment by fbastian** [comment]
Don't start this until the problems with RNA-Seq are completly fixed.

**Comment by mimiblis** [comment]
Anyway not ;-) because the cleaning of rnaseq problems May have an influence on affy
